src/common/components/repository-row/index.js

```diff
@@ -19,9 +21,12 @@ const RepositoryRow = (props) => {
             and also show 'Never built' when no builds available
         */}
         { latestBuild &&
-          <Link to={ `/${fullName}/builds/${latestBuild.buildId}`}>
-            { latestBuild.statusMessage }
-          </Link>
+          <BuildStatus
```

Comment:
It would be great to have more react components tests - but as repos list doesn't have any yet, I don't want to spend time creating them from scratch now.